client/src/legacy/LeftAndMain.js

```diff
@@ -628,7 +615,17 @@ $.entwine('ss', function($) {
         headers: headers,
         url: historyState.path || document.URL
       })
+      .fail((xhr, status, error) => {
+        // Ignoring aborts: The server request failed, so prevent a mixed UI state by rolling back to previously
+        // succesfully loaded URL (consistent with currently loaded content).
+        if (xhr.readyState !== 0) {
```

Comment:
why do we need to test for `xhr.readyState !== 0`?

Reply:
I noticed that when switching between page settings tabs, the XHR was requested multiple times but the history state was updated only once. The first request was nearly immediately aborted (resulting in `.readyState` of 0).

Reply:
p.s. While I happened to stumble upon this buggy side effect (due to inconsistent/repeated XHR's) I noticed that pushing this line also fixed some [failing tests](https://app.travis-ci.com/github/silverstripe/silverstripe-admin/jobs/554830938#L1751) as well. So, I guess it didn't just affect the page settings/history tabs.